### ISSUE TITLE
Feature Request: Notification for added media

### ISSUE THREAD

**Issue by JustACasual** [jellyfin/jellyfin-android#1490]
Would it be possible to make the app show a notification when new media is added?
Of course, more settings would be great, make users subscribe to certain types of media, and then certain items. But making this the same as webhooks settings, would already be nice as a first step.

**Comment by BotBlake** [comment]
@mdPlusPlus Yes there is a way to send Webhooks directly from the jellyfin server whenever changes happen.
For this you can use the https://github.com/jellyfin/jellyfin-plugin-webhook Plugin.
Hope this helps.
